packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -232,17 +244,30 @@ export async function init({ appUrls, flex }: { appUrls?: string | null; flex?:
         await registerComponentDependencyPaths(JSON.parse(appUrls));
     }
 
+    // Load custom initialization module
+    if (customInit) {
+        await sap.ui.require([customInit]);
```

Comment:
Is the promise returning version of `sap.ui.require` available in UI5 `1.71.x`? That's the oldest version we support. If not, please use a callback like in line 232

Reply:
Seems [to be supported](https://sapui5.hana.ondemand.com/1.71.64/#/api/sap.ui/methods/sap.ui.require) 😅

Reply:
I'm asking myself if we need to `await` this `sap.ui.require` at all. In difference to line 232 this is just an additional module that should be loaded. So far nobody is interested in any return value for further processing. So we could simply go for `sap.ui.require([customInit]);`

Reply:
Agree, there is no need to wait, maybe we could log something if a custom script is loaded but not really